Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -236,7 +236,7 @@ extension ProjectDescription.Target {
             return nil
         }
 
-        guard let product = ProjectDescription.Product.from(name: target.name, products: products, productTypes: productTypes) else {
+        guard let product = ProjectDescription.Product.from(name: target.name, hasResources: !target.resources.isEmpty, products: products, productTypes: productTypes) else {
```

Comment:
The logic to map resources is already in `ResourceFileElements.from`, to make sure we are aligned here, I would use:
```suggestion
        let resources = ResourceFileElements.from(resources: target.resources, path: path)
        guard let product = ProjectDescription.Product.from(name: target.name, hasResources: resources != nil, products: products, productTypes: productTypes) else {
```